test/OpenTelemetry.Tests/Metrics/MetricSnapshotTestsBase.cs

```diff
@@ -16,10 +16,9 @@ public abstract class MetricSnapshotTestsBase
 {
     private readonly IConfiguration configuration;
 
-    protected MetricSnapshotTestsBase(bool shouldReclaimUnusedMetricPoints)
+    protected MetricSnapshotTestsBase()
```

Comment:
@xiang17, can this class be fully removed and replaced by MetricSnapshotTests without any inheritance?

Reply:
#5960